I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcublas.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcudnn.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcufft.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcurand.so locally
I tensorflow/core/common_runtime/gpu/gpu_init.cc:102] Found device 0 with properties: 
name: Tesla K80
major: 3 minor: 7 memoryClockRate (GHz) 0.8235
pciBusID 0000:1a:00.0
Total memory: 11.25GiB
Free memory: 11.16GiB
I tensorflow/core/common_runtime/gpu/gpu_init.cc:126] DMA: 0 
I tensorflow/core/common_runtime/gpu/gpu_init.cc:136] 0:   Y 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:838] Creating TensorFlow device (/gpu:0) -> (device: 0, name: Tesla K80, pci bus id: 0000:1a:00.0)
I tensorflow/core/common_runtime/gpu/pool_allocator.cc:244] PoolAllocator: After 3910 get requests, put_count=3890 evicted_count=1000 eviction_rate=0.257069 and unsatisfied allocation rate=0.286445
I tensorflow/core/common_runtime/gpu/pool_allocator.cc:256] Raising pool_size_limit_ from 100 to 110
Using TensorFlow backend.
/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/Keras-1.1.0-py3.4.egg/keras/backend/tensorflow_backend.py:1057: UserWarning: Expected no kwargs, you passed 1
kwargs passed to function are ignored with Tensorflow backend
  warnings.warn('\n'.join(msg))
Traceback (most recent call last):
  File "./train.py", line 145, in <module>
    run(epochs=epochs,batch_size=batch_size,mid=mid)
  File "./train.py", line 126, in run
    verbose=1)
  File "/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/Keras-1.1.0-py3.4.egg/keras/models.py", line 874, in fit_generator
    pickle_safe=pickle_safe)
  File "/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/Keras-1.1.0-py3.4.egg/keras/engine/training.py", line 1456, in fit_generator
    class_weight=class_weight)
  File "/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/Keras-1.1.0-py3.4.egg/keras/engine/training.py", line 1234, in train_on_batch
    outputs = self.train_function(ins)
  File "/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/Keras-1.1.0-py3.4.egg/keras/backend/tensorflow_backend.py", line 1040, in __call__
    updated = session.run(self.outputs + [self.updates_op], feed_dict=feed_dict)
  File "/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/tensorflow/python/client/session.py", line 710, in run
    run_metadata_ptr)
  File "/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/tensorflow/python/client/session.py", line 908, in _run
    feed_dict_string, options, run_metadata)
  File "/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/tensorflow/python/client/session.py", line 958, in _do_run
    target_list, options, run_metadata)
  File "/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/tensorflow/python/client/session.py", line 965, in _do_call
    return fn(*args)
  File "/dccstor/dlw/virtual_environments/python3.4/lib/python3.4/site-packages/tensorflow/python/client/session.py", line 947, in _run_fn
    status, run_metadata)
KeyboardInterrupt
